src/scdef/models/_scdef.py

```diff
@@ -2040,24 +2054,73 @@ def plot_gini_brd(
         is_kept = np.zeros((self.layer_sizes[0]))
         is_kept[self.factor_lists[0]] = 1
 
-        plt.figure(figsize=figsize)
+        if ax is None:
+            fig, ax = plt.subplots(figsize=figsize)
+
         for c in [1, 0]:
             label = "Kept" if c == 1 else "Removed"
             color = "C0" if c == 1 else "gray"
             _alpha = 1 if c == 1 else alpha
-            plt.scatter(
+            ax.scatter(
                 ginis[np.where(is_kept == c)[0]],
                 brds[np.where(is_kept == c)[0]],
                 label=label,
                 color=color,
                 alpha=_alpha,
             )
-        plt.xlabel("Gini index", fontsize=fontsize)
-        plt.ylabel("BRD posterior mean", fontsize=fontsize)
-        plt.legend(fontsize=legend_fontsize)
+        ax.set_xlabel("Gini index", fontsize=fontsize)
+        ax.set_ylabel("BRD posterior mean", fontsize=fontsize)
+        ax.legend(fontsize=legend_fontsize)
+
+        if show:
+            plt.show()
+        else:
+            return ax
 
+    def plot_likelihood(self, figsize=(4, 4), fontsize=12, ax=None, show=True):
+        if ax is None:
+            fig, ax = plt.subplots(figsize=figsize)
+        ax.plot(np.concatenate(self.elbos)[:])
+        ax.set_yscale("log")
+        ax.set_xlabel("# ELBO", fontsize=fontsize)
+        ax.set_ylabel("Likelihood [log]", fontsize=fontsize)
```

Comment:
Not sure I agree with the nomenclature here. Would you consider renaming to `plot_elbos`? And then `ax.set_xlabel('iteration')`, `ax.set_ylabel('ELBO [log]')`. The ELBO is an approximation to the marginal likelihood, but it's not the marginal likelihood itself

Reply:
changed to `Loss [log]` vs. `Epoch`